### PR TITLE
update config project for using as nuget

### PR DESCRIPTION
**Pull request by Oleg-Mozhey**

### CODE REVIEW

**Comment by Oleg-Mozhey** [comment]
How to use this Configuration project:
1. Build nuget package based on the configuration project
2. Place the package into local storage
3. Import package in your project
4. Setup path to configuration json in hooks.

Example of path configuration:
    [BeforeTestRun]
    public static void ConfigureRunMode()
    {
        var env = Environment.GetEnvironmentVariable("STAND");
        
        if (string.IsNullOrEmpty(env))
        {
            var attributes = Assembly.GetExecutingAssembly().GetCustomAttributes();
            var configAttribute = attributes.FirstOrDefault(attribute => attribute.TypeId.Equals(typeof(AssemblyConfigurationAttribute)));
            var configEnv = (AssemblyConfigurationAttribute) configAttribute!;
            env = configEnv.Configuration;
        }
        ConfigManager.InitConfiguration(env);
    }

File: Behavioral.Automation.Configs/ConfigManager.cs [hunk 1]
```diff
@@ -1,24 +1,13 @@
...
 
 public static class ConfigManager
 {
-    private static IConfiguration Configuration { get; }
+    private static IConfiguration Configuration { get; set; }
 
-    static ConfigManager()
+    public static void InitConfiguration(string env)
     {
-        var env = Environment.GetEnvironmentVariable("STAND");
```
Comment (andrey-brovko) [inline Behavioral.Automation.Configs/ConfigManager.cs:12]: Let's leave the STAND variable.

**Comment by Oleg-Mozhey** [comment]
How to use this Configuration project:
1. Build nuget package based on the configuration project
2. Place the package into local storage (I hope one day this will be available in the remote repo)
3. Import package in your project
4. Setup environment variable "STAND" according to the stand for testing .

Example of "STAND" variable configuration:
```
   [BeforeTestRun]
    public static void ConfigureRunMode()
    {
        var attributes = Assembly.GetExecutingAssembly().GetCustomAttributes();
        var configAttribute = attributes.FirstOrDefault(attribute => attribute.TypeId.Equals(typeof(AssemblyConfigurationAttribute)));
        var configEnv = (AssemblyConfigurationAttribute) configAttribute!;
        var env = configEnv.Configuration;
        Environment.SetEnvironmentVariable("STAND", env);
    }
```

File: Behavioral.Automation.Configs/ConfigManager.cs (hunk 1 above)
Comment (Oleg-Mozhey) [reply Behavioral.Automation.Configs/ConfigManager.cs:12]: Ok

**Review by ILastovka — APPROVED** [review]

File: Behavioral.Automation.Configs/ConfigManager.cs [hunk 2]
```diff
@@ -9,15 +8,7 @@ public static class ConfigManager
...
-        
-        if (string.IsNullOrEmpty(env))
-        {
-            var attributes = Assembly.GetExecutingAssembly().GetCustomAttributes();
-            var configAttribute = attributes.FirstOrDefault(attribute => attribute.TypeId.Equals(typeof(AssemblyConfigurationAttribute)));
-            var configEnv = (AssemblyConfigurationAttribute) configAttribute!;
-            env = configEnv.Configuration;
-        }
+        var env = Environment.GetEnvironmentVariable("STAND")?? "";
```
[the remark this answers is not in the document]
Comment (Oleg-Mozhey) [reply Behavioral.Automation.Configs/ConfigManager.cs:11]: It's used in the following code
image

File: Behavioral.Automation.Configs/ConfigManager.cs [hunk 3]
```diff
@@ -9,15 +8,7 @@ public static class ConfigManager
...
 
         Configuration = new ConfigurationBuilder()
             .AddJsonFile("AutomationConfig.json", true, true)
```
Comment (andrey-brovko) [inline Behavioral.Automation.Configs/ConfigManager.cs:14]: Maybe add something like this?
```
if(string.IsNullOrWhiteSpace(env)
{ 
Configuration.AddJsonFile($"AutomationConfig.{env.ToLower()}.json", true, true)
}
```

**Pull request merged**